fractal_server/app/api/v1/monitoring.py

```diff
@@ -0,0 +1,141 @@
+from datetime import datetime as DateTime
+from typing import Optional
+
+from fastapi import APIRouter
+from fastapi import Depends
+from sqlmodel import select
+
+from ...db import AsyncSession
+from ...db import get_db
+from ...models import ApplyWorkflow
+from ...models import Dataset
+from ...models import Project
+from ...models import Workflow
+from ...schemas import ApplyWorkflowRead
+from ...schemas import ProjectRead
+from ...security import current_active_superuser
+from ...security import User
+from fractal_server.app.schemas.workflow import WorkflowTaskStatusType
+
+router = APIRouter()
+
+
+@router.get("/project/")
```

Comment:
Here, and below, let's include `response_model` (and make sure that it matches with the return type)